### PR TITLE
[New Rule] Lateral Movement via Kerberos using Bifrost Console

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/842

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/macos/lateral_movement_credential_access_kerberos_bifrostconsole.toml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+[metadata]
+creation_date = "2020/01/12"
+maturity = "production"
+updated_date = "2020/01/12"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the use of Bifrost Console, a known macOS Kerberos penstesting tool that can be used to dump cached Kerberos
+tickets, launch pass the tickets and hash, kerberoasting and other related techniques.
```
Comment (peasead) [inline rules/macos/lateral_movement_credential_access_kerberos_bifrostconsole.toml:10]:
```suggestion
Identifies the use of the Bifrost Console, a known macOS Kerberos pentesting tool that can be used to dump cached Kerberos
tickets or attempt unauthorized authentication techniques such as pass-the-ticket/hash and kerberoasting.
```

File: rules/macos/lateral_movement_credential_access_kerberos_bifrostconsole.toml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+name = "Lateral Movement via Kerberos using Bifrost Console"
+references = ["https://github.com/its-a-feature/bifrost"]
+risk_score = 73
+rule_id = "16904215-2c95-4ac8-bf5c-12354e047192"
+severity = "high"
+tags = ["Elastic", "Host", "macOS", "Threat Detection", "Credential Access", "Lateral Movement"]
+type = "query"
+
+query = '''
+event.category:process and event.type:(start or process_started) and
```
Comment (peasead) [inline rules/macos/lateral_movement_credential_access_kerberos_bifrostconsole.toml:25]:
```suggestion
event.category:process and event.type:start and 
```

File: rules/macos/lateral_movement_credential_access_kerberos_bifrostconsole.toml [hunk 3]
```diff
@@ -0,0 +1,55 @@
+[metadata]
+creation_date = "2020/01/12"
+maturity = "production"
+updated_date = "2020/01/12"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the use of the Bifrost Console, a known macOS Kerberos pentesting tool that can be used to dump cached Kerberos
+tickets or attempt unauthorized authentication techniques such as pass-the-ticket/hash and kerberoasting.
```
Comment (bm11100) [inline rules/macos/lateral_movement_credential_access_kerberos_bifrostconsole.toml:10]:
```suggestion
Identifies use of Bifrost, a known macOS Kerberos pentesting tool, which can be used to dump cached Kerberos
tickets or attempt unauthorized authentication techniques such as pass-the-ticket/hash and kerberoasting.
```

File: rules/macos/lateral_movement_credential_access_kerberos_bifrostconsole.toml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+"""
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Lateral Movement via Kerberos using Bifrost Console"
```
Comment (bm11100) [inline rules/macos/lateral_movement_credential_access_kerberos_bifrostconsole.toml:16]:
```suggestion
name = "Potential Kerberos Attack via Bifrost"
```

**Review by bm11100 — APPROVED** [review]

**Review by brokensound77 — APPROVED** [review]
🚢

**Pull request merged**